### PR TITLE
install fastchardet from PyPi

### PR DESCRIPTION
**Pull request by myrdd**
When calling `setup.py`, fastchardet is not installed, because `-e` is excluded. However, fastchardet is available from PyPi, so I suggest to use that.

Another solution (or an additional one) is to remove the exception for `-e` in `setup.py`, see https://github.com/mozilla/amo-validator/blob/v1.7.0/setup.py#L18.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -4,5 +4,5 @@ Fabric==0.9.0
 nose==1.0.0
 rdflib==3.2.1
 argparse==1.1
--e git://github.com/mattbasta/fastchardet#egg=fastchardet
+fastchardet
```
Comment (magopian) [inline requirements.txt]: Please pin the version number: `fastchardet==0.2.0` (which is the latest version on pypi, and seems to also be the last commit on `master`, the one we've been using with the `-e` option)

**Comment by magopian** [comment]
Excellent catch! Thanks a lot for that, very useful.

r+wc

P.S: `r+wc` means
1/ `r+`: reviewed, and approved (in opposition to `r-`)
2/ `wc`: with comments (with some comments or fixes needed)
Answering the comments should mean that this PR is ready to be merged!

File: requirements.txt (hunk 1 above)
Comment (myrdd) [inline requirements.txt]: Thanks for the quick reply! I'll do that. However, do you use to amend the changes and/or squash?

Comment (magopian) [inline requirements.txt]: one or the other, as you prefer. For big changes, creating a new commit is easier to review, and then we squash before merging. For such a small change, amending is fine ;)

Comment (myrdd) [inline requirements.txt]: Okay, but since I didn't want to force-push, I created a new commit.

Comment (magopian) [inline requirements.txt]: It's also good. Amending or squashing will always need a force push. I can do the squashing myself when merging, which i'm going to do know. Thanks!

**Comment by magopian** [comment]
Merged in https://github.com/mozilla/amo-validator/commit/f3c6b9d60133453a61141f3c341609a4d0efc21c

**Comment by magopian** [comment]
Thanks @myrdd !

**Pull request closed without merging**